src/test/kotlin/it/pagopa/wallet/services/WalletServiceTest.kt

```diff
@@ -3599,4 +3655,55 @@ class WalletServiceTest {
             }
             .verifyComplete()
     }
+
+    @Test
+    fun `should throw InvalidArgument error when creation session for invalid Paypal pspId`() {
```

Comment:
If I'm reading this correctly, this test case tests an empty response (no bundles) from GEC. If that's the case I would map this condition to a different HTTP status code (e.g. 422 or 404 rather than 400) or to a different exception altogether: in this case the user request is formally valid but we can't process it correctly

Reply:
Yes, I'm agree with 404 with a detailed message, I will add. What do you think?

Reply:
Solved by https://github.com/pagopa/pagopa-wallet-service/pull/144/commits/49c29495c8ef1532247d5068c7f70242ad5095ca